src/njoy21/input/VIEWR/Card1/Nps.hpp

```diff
@@ -0,0 +1,28 @@
+struct Nps { 
```

Comment:
Yeah, that would cause a problem. I guess you do have to override the `verify` function. Sorry.